tensorflow/lite/kernels/atan2.cc

```diff
@@ -40,33 +38,31 @@ TfLiteStatus Atan2Prepare(TfLiteContext* context, TfLiteNode* node) {
   const TfLiteTensor* input_y = tflite::GetInput(context, node, 0);
   const TfLiteTensor* input_x = tflite::GetInput(context, node, 1);
   TfLiteTensor* output = tflite::GetOutput(context, node, 0);
-
   // Validate size and type constraints
   TF_LITE_ENSURE_OK(context, EnsureSameShape(context, input_y, input_x));
   TF_LITE_ENSURE_TYPES_EQ(context, input_y->type, input_x->type);
   TF_LITE_ENSURE_TYPES_EQ(context, input_y->type, output->type);
-  TF_LITE_ENSURE(context,
-                 input_y->type == kTfLiteFloat32 ||
-                 input_y->type == kTfLiteFloat64);
+  TF_LITE_ENSURE(context, input_y->type == kTfLiteFloat32 ||
```

Comment:
Please make sure to run clang-format on this file, these seem misaligned.

Reply:
Okay